### PR TITLE
Peformance test tools.

### PR DESCRIPTION
**Pull request by shenning00**
Added qperf2 - qperf_pub and qperf_sub test tools. These tools are part of the 1:many test scenario.

### CODE REVIEW

File: cmd/qperf2/qperf.hpp [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+        /*
+        std::uint64_t start_transmit_time;
+        std::uint64_t end_transmit_time;
+        std::uint64_t total_published_objects;
+        std::uint64_t total_published_bytes;
+        std::uint64_t max_bitrate;
+        std::uint64_t min_bitrate;
+        std::uint64_t avg_bitrate;
+        */
+    };
```
Comment (GhostofCookie) [inline cmd/qperf2/qperf.hpp:71]: Are these going to be needed?

File: cmd/qperf2/qperf.hpp [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+
+    inline quicr::FullTrackName MakeFullTrackName(const std::string& track_namespace,
+                                                  const std::string& track_name,
+                                                  const std::optional<uint64_t> track_alias = std::nullopt) noexcept
+    {
+        return {
+            quicr::TrackNamespace{ quicr::Bytes{ track_namespace.begin(), track_namespace.end() } },
```
Comment (GhostofCookie) [inline cmd/qperf2/qperf.hpp:78]: TrackNamespace can take strings as well, shouldn't have to use Bytes here.

File: include/quicr/detail/priority_queue.h [hunk 3]
```diff
@@ -203,6 +203,7 @@ namespace quicr {
         std::unique_ptr<TimeQueueType>& GetQueueByPriority(const uint8_t priority)
         {
             if (priority >= PMAX) {
+                std::cerr << "PMAX " << (int)PMAX << " and p = " << (int)priority << std::endl;
```
Comment (GhostofCookie) [inline include/quicr/detail/priority_queue.h:206]: Do we want to keep this here?

**Review by TimEvens — APPROVED** [review]
LGTM to get this in.

**Comment by shenning00** [comment]
Merging for now. Will replace qperf with qperf2 in the future.

**Pull request merged**